src/parquet/file/writer.h

```diff
@@ -152,19 +143,9 @@ class PARQUET_EXPORT ParquetFileWriter {
    */
   const std::shared_ptr<WriterProperties>& properties() const;
 
-  /**
-   * Returns the file schema descriptor
-   */
-  const SchemaDescriptor* descr() { return schema_; }
```

Comment:
These two function are quite comfortable, I'd like to preserve them. It's ok for me if they call a bit deeper but they are just convenient from an API user level.